sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/models/InternalError.java

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
```

Comment:
If Public class shouldn't have a setter.

Reply:
Refer to PersonalizationCustomization, line 21 on why we had to make an exception here. Exception was granted since this is in beta.